src/test/integration/IntegrationBase.t.sol

```diff
@@ -14,7 +14,7 @@ import "src/test/integration/TimeMachine.t.sol";
 import "src/test/integration/users/User.t.sol";
 import "src/test/integration/users/User_M1.t.sol";
 
-abstract contract IntegrationBase is IntegrationDeployer {
+abstract contract IntegrationBase is IntegrationDeployer, IAllocationManagerTypes {
```

Comment:
Can we remove IAllocationManagerTypes

Reply:
fixed in [d7a5d61](https://github.com/Layr-Labs/eigenlayer-contracts/pull/934/commits/d7a5d61437737a470dd4d22220e8a818d21c2ac1)